test/ui-testing/error-messages.js

```diff
@@ -72,8 +72,11 @@ module.exports.test = function uiTest(uiTestCtx) {
           .wait('#section-patron div[class*="Error"]')
           .evaluate(() => {
             const errorText = document.querySelector('#section-patron div[class*="Error"]').innerText;
-            if (!errorText.startsWith('User')) {
-              throw new Error('Error message not found for invalid user input');
+            if (!errorText.startsWith('User with this')) {
+              throw new Error(`Error message not found for invalid user input: ${errorText}`);
```

Comment:
@zburke, do not be hard on yourself, such things happen. The only comment from me is the following.

I see that this line is the same and inside the `if` below. I suggest combining both conditions in one then.

`if (!errorText.startsWith('User with this') || !errorText.endsWith('does not exist')) {`

Reply:
Good call.